scripts/statesync-pebble.bash

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# microtick and bitcanna contributed significantly here.
+# Pebbledb state sync script.
+set -uxe
+
+# Set Golang environment variables.
+export GOPATH=~/go
+export PATH=$PATH:~/go/bin
```

Comment:
I think we can comment these lines by default assuming they are already set

Reply:
nice